<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Campo:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="campoCb"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Chefe VTR:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="chefeVtrLe"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Motorista:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="motoristaLe"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Placa VTR:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="placaVtrLe"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="diaLe">
     <property name="placeholderText">
      <string>AAAA-MM-DD</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Hora In√≠cio:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inicioLe">
     <property name="placeholderText">
      <string>HH:MM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Hora Fim:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fimLe">
     <property name="placeholderText">
      <string>HH:MM</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
